portal/course.py

```diff
@@ -81,7 +81,7 @@ def edit(id):
         g.db.commit()
         cur.close()
 
-        return redirect(url_for('portal.home'))
+        return redirect(url_for('course.home'))
```

Comment:
Joey was working on breaking up all of the code into modules, and i think hes using course.py for the course stuff, lol- ugh, sorry.... maybe we can use main.py, since its the main module? we can talk to him about it in our standup.... other then that your tests and everything looks really good